app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+def create_sorted_variables(*args) -> dict:
+    sorted_variables = {"mutable": [], "immutable": []}
+
+    for arg in args:
+        if isinstance(arg, (list, set, dict)):
+            sorted_variables["mutable"].append(arg)
+        else:
+            sorted_variables["immutable"].append(arg)
+
+    return sorted_variables
+
+
+sorted_variables = create_sorted_variables(lucky_number, pi, one_is_a_prime_number, name, my_favourite_films, profile_info, marks, collection_of_coins)
```

Comment:
The 'sorted_variables' variable is not used anywhere in your code. If you don't plan to use it later, consider removing it to keep your code clean and efficient.